src/aind_data_schema_models/mouse_anatomy_ontology.py

```diff
@@ -0,0 +1,23 @@
+"""Module for Mouse Anatomy"""
+
+from importlib_resources import files
+from pydantic import BaseModel, ConfigDict, Field
+
+from aind_data_schema_models.utils import create_literal_class, read_csv
+
+
+class MouseAnatomyModel(BaseModel):
```

Comment:
I think we want this class to mirror our PIDName structure. (PIDName stands for Persistent Identifier Name).
It's possible with the re-org of this repo, we don't need to use the PIDName class itself, but I think we want to keep the fields.

name: 
abbreviation: (we probably don't have this for this ontology, so we can possibly leave this out)
registry: Edinburgh Mouse Atlas Project
registry_identifier: (and here we need the value to include the "EMAPA" label along with the id #. This is because there are two versions: EMAP and EMAPA and we're using the latter.

Reply:
@dyf do you have opinions on leaving out abbreviation?

Reply:
yeah, let's leave out abbreviation here.

Reply:
ah, wait, we want to use the Registry model for the registry field. I think I already added this registry to that